packages/fhir-response-util/index.js

```diff
@@ -11,6 +11,7 @@ function getContentType(version) {
       return 'application/json+fhir';
     case '3_0_1':
     case '4_0_0':
+    case '4_0_1':
```

Comment:
This will work.  But we can make this better.
Why not make 3 arrays - json_fhir, fhir_json, and json.  Or a map where key = version and value = content type.
Then we can simply add 4_0_2.  And the unit test can also just pick up the new value.
The base version can also be the latest version

Reply:
Can you expand on this comment a little bit? I sketched out a solution as follows, but I don't see how we can modify the unit tests to be dynamic yet have them not assert a tautology:

``` Javascript
const contentTypeMap = {
  '1_0_2': 'application/json+fhir',
  '3_0_1': 'application/fhir+json',
  '4_0_0': 'application/fhir+json',
  '4_0_1': 'application/fhir+json',
};

function getContentType(version) {
  return contentTypeMap[version] || 'application/json';
}
```

Assume I updated the version to be a lookup object like above. In the unit tests, we'd still need to know what the expected Content-Type headers are for specific versions, right?

Reply:
that looks good.  see comment for index.test.js